hack/update-codecgen.sh

```diff
@@ -144,7 +144,7 @@ for current in "${index[@]}"; do
   # We use '-d 1234' flag to have a deterministic output every time.
   # The constant was just randomly chosen.
   ###echo "DBG: running ${CODECGEN} -d 1234 -o ${base_generated_file} ${base_file}"
-  ${CODECGEN} -d 1234 -o "${base_generated_file}" "${base_file}"
+  ${CODECGEN} -d 1234 -o "${base_generated_file}" -nr 'ObjectMeta' "${base_file}"
```

Comment:
Making codecgen multi-package aware fixes the problem at hand. But it makes codec generation uncomposable, i.e. downstream has to remove and regenerate vendored codec files.